doc/README.rst

```diff
@@ -1,24 +1,3 @@
 .. _contributing.docs:
```

Comment:
Do you mind also removing this? I don't think we want to link to this README (and probably we can't as this is not in the sphinx sources dir). And it's confusing to have it here.